cli/src/lib/git.js

```diff
@@ -1,8 +1,33 @@
 // @flow
-
 import whichCb from 'which';
+import simpleGit from 'simple-git';
+
 import {child_process} from './node';
 
+const mainlineBranch = 'main';
```

Comment:
this was more so that I didn't need to repeat `main` keyword all over the codebase. But it's quite locked in.

How do you mean by config though? Should I shift this var to another utility module do you think?